docs/introduction/getting-started.mdx

```diff
@@ -175,12 +175,18 @@ main();
 
 ### Test and confirm
 
-It's time to run your app! Because the Superface OneSDK is built with Wasm, it requires that you pass the `--experimental-wasi-unstable-preview1` flag when running Node.js (>18.0.0).
+It's time to run your app! Because the Superface OneSDK is built with Wasm, it requires that you pass the `--experimental-wasi-unstable-preview1` flag when running Node.js version < `18.17.0`.
```

Comment:
Change itself is fine, but the flag is needed, because we are using [WASI](https://wasi.dev) for WebAssembly.

```suggestion
It's time to run your app! Because the Superface OneSDK is built with WebAssembly and uses WASI, it requires that you pass the `--experimental-wasi-unstable-preview1` flag when running Node.js version < `18.17.0`.
```